bonusopdracht.js

```diff
@@ -0,0 +1,64 @@
+//1 + 2
+function sortTvsOnPrice (tvArray) {
+    const sortedTvs = inventory.sort((a, b) => a.price - b.price);
+    console.log(sortedTvs);
+
+    const tv = document.createElement('p');
+    tv.textContent = showAllTvs(sortedTvs);
+    const tvsOnPage = document.getElementById('sort-tvs-on-price');
+    tvsOnPage.appendChild(tv);
+};
+
+function showAllAmbilightTvs(tvArray) {
+    const ambilightTvs = tvArray.filter((tv) => {
+        return tv.options.ambiLight === true;
+    })
```

Comment:
Goed gedaan! Je kan dit nog korter schrijven:
```suggestion
    const ambilightTvs = tvArray.filter(tv => tv.options.ambiLight)
```